osc-staging.py

```diff
@@ -229,6 +230,7 @@ def do_staging(self, subcmd, opts, *args):
         osc staging unlock
         osc staging rebuild [STAGING...]
         osc staging repair REQUEST...
+        osc staging setprio [LETTER...]
```

Comment:
Presumably this works with adi stagings since it ends up calling `staging_projects` which works for adi. As such perhaps stick with `[STAGING...]`. Could also use `stagingapi.prj_from_short()` to allow for full project name in addition to short hand.